src/main/java/com/cleanroommc/groovyscript/compat/mods/jei/JustEnoughItems.java

```diff
@@ -35,10 +50,64 @@ public void removeAndHide(IIngredient ingredient) {
         JeiPlugin.hideItem(ingredient.getMatchingStacks());
     }
 
+    public void removeAndHide(IIngredient... ingredients) {
+        for (IIngredient ingredient : ingredients) {
+            if (IngredientHelper.isEmpty(ingredient)) {
+                GroovyLog.msg("Error remove and hide items {}", ingredient)
+                        .add("Items must not be empty")
+                        .error()
+                        .post();
+                return;
+            }
+            JeiPlugin.hideItem(ingredient.getMatchingStacks());
+        }
+        for (IRecipe recipe : ForgeRegistries.RECIPES) {
+            if (recipe.getRegistryName() != null) {
+                for (IIngredient ingredient : ingredients) {
+                    if (ingredient.test(recipe.getRecipeOutput())) {
+                        ReloadableRegistryManager.removeRegistryEntry(ForgeRegistries.RECIPES, recipe.getRegistryName());
```

Comment:
Why does it use a different method than the single param method?

Reply:
To avoid multiple iterations of the recipe registry